pyproject.toml

```diff
@@ -6,11 +6,90 @@ requires = [
   "wheel",
 ]
 
+[project]
+name = "compliance-checker"
+description = "Checks Datasets and SOS endpoints for standards compliance"
+readme = "README.md"
+license = {text = "Apache-2.0"}
+maintainers = [
+    {name = "Dave Foster", email = "dave@axiomdatascience.com"},
+    {name = "Benjamin Adams"},
+    {name = "Luke Campbell"},
+    {name = "Filipe Fernandes"},
+]
+requires-python = ">=3.8"
+classifiers=[
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: POSIX :: Linux",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Scientific/Engineering",
+]
+dynamic = [
+  "dependencies",
+  "version",
+]
+[project.urls]
+documentation = "https://ioos.github.io/compliance-checker"
+homepage = "https://compliance.ioos.us/index.html"
+repository = "https://github.com/ioos/compliance-checker"
+[project.scripts]
+compliance-checker = "cchecker:main"
```

Comment:
This is deprecated and in the future. We need a better solution to ship that script.